starter-code/client/src/pages/RidesList.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import axios from 'axios';
+import styled from "styled-components";
+
+import getUser from "../utils/get-user";
+import Layout from "../components/Layout";
+import Container from "react-bootstrap/Container";
+import ListObject from "./ride-files/ListObject";
+
+
+const TextWrapper = styled.div`
+  width: 700px;
+  max-width: 100%;
+`;
```

Comment:
Should we keep css stuff out of the js files and keep it in separate .css files that we can import?